<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteGapsDialog</class>
 <widget class="QDialog" name="DeleteGapsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove columns of gaps</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="absoluteRadioButton">
       <property name="text">
        <string>Remove columns with number of gaps</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="absoluteSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="relativeRadioButton">
       <property name="text">
        <string>Remove columns with percentage of gaps</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="relativeSpinBox">
       <property name="suffix">
        <string comment="%">%</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="allRadioButton">
       <property name="text">
        <string>Remove all gap-only columns</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeleteGapsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeleteGapsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
